doc/CSV_and_XML.rst

```diff
@@ -0,0 +1,62 @@
+.. py:currentmodule:: lsst.ts.hvac
+
+.. _lsst.ts.hvac-CSV_and_XML:
+
+###########
+CSV and XML
+###########
+
+CSV
+---
+
+The Rubin Observatory HVAC system was implemented by DatControl who also provided an Excel sheet with the full information about the HVAC system.
+This includes the names of the HVAC components and the data type, limits and units of the telemetry published.
+This information is necessary for validation purposes since the `MQTT`_ protocol doesn't provide that information.
+The individual sheets in the Excel sheet have manually been merged into a single sheet which was exported to CSV.
+This CSV file can be found in the python/data directory and serves as input for several operations:
+
+* To read all MQTT topics
+* To read the data types, limits and units for all MQTT topics
+
+The data read from the CSV file are kept in memory and are used for all operations provided by the ts_hvac project:
+
+* Create the XML files describing the Commands, Events and Telemetry which get used in ts_xml.
+* Verify the received MQTT messages by the HVAC CSC and compute medians of the received telemetry.
+* Simulate the publication of HVAC telemetry and the reception of HVAC commands by the simulator.
+* Unit testing of the CSC and simulator.
+
+The columns in the CSV file are:
+
+* floor
+* subsystem
+* variable
+* topic_and_item
+* publication
+* subscription
+* signal
+* rw
+* range
+* limits
+* unit
+* state
+* observations
+* notes
+
+where "rw" means Read or Write.
+
+.. _MQTT: ./MQTT.html
```

Comment:
Recommendation is to use 
```
:doc:`MQTT`
``` 
Which as part of the build generates a hyperlink to the generated html instead of referring to the html page directly.

Reply:
That modification creates a warning when the documentation is created but the result still works. Thanks!

Reply:
What's the warning?

Reply:
The warning disappeared when I renamed MQTT.rst to mqtt.rst.